ONSdigital/census-rm-docker-dev
# Upgrade Postgres to v11

# Motivation and Context

Upgrade our DB to use v11

# What has changed



Changed to use the v11 image
Added password for Docker run command (we get errors if this isn't set)

# How to test?



`make down && make up`
Run the ATs to make sure everything is wired up correctly

# Links



https://trello.com/c/srTKq51H/624-upgrade-to-postgres-11-2
